controllers/admin/AdminImagesController.php

```diff
@@ -178,6 +192,51 @@ public function __construct()
                 ],
                 'submit' => ['title' => $this->trans('Save', [], 'Admin.Actions')],
             ],
+            'optimization' => [
+                'title' => $this->trans('Advanced optimization options', [], 'Admin.Design.Feature'),
+                'icon' => 'icon-picture',
+                'top' => '',
+                'bottom' => '',
+                'description' => $this->trans('Optimize bandwidth and disk usage.', [], 'Admin.Design.Help'),
+                'fields' => [
+                    'PS_IMAGEOPT_NO_ENLARGE' => [
+                        'type' => 'bool',
+                        'title' => $this->trans('Never create images larger than the base image', [], 'Admin.Design.Feature'),
+                        'required' => false,
+                        'is_bool' => true,
+                        'hint' => $this->trans('Never add a white or transparent border around an image.', [], 'Admin.Design.Help'),
+                        'desc' => $this->trans('Optimizes bandwidth and disk usage, adds flexibility to template, resizing is handled by the browser. May not be compatible with all templates.', [], 'Admin.Design.Help'),
```

Comment:
Adds flexibility to the template means this:
When images get upsized with a border around, this is a very heavy limitation on the template: the display size of the image MUST always be identical to the pixel size of the image. This is a MUST, because of this scenario:
- original file size 200x200px
- thumbnail size 300x300px (anything, but larger). In Prestashop default, the thumbnail will have a size of 300x300, but with a border of 50px around the original image of 200px
- On the page, the displayed image has a size of 150px (this could be for two reasons (1) high definition display (2) responsive image resizing) 
With Prestashop default behavior, we will now see an image of 100px with a white border of 25px around. 
With this setting enabled, we will se the image taking all the 150px available.

Reply:
@Julievrz 
Adds flexibility to the template means this:
When images get upsized with a border around, this is a very heavy limitation on the template: the display size of the image MUST always be identical to the pixel size of the image. This is a MUST, because of this scenario:
- original file size 200x200px
- thumbnail size 300x300px (anything, but larger). In Prestashop default, the thumbnail will have a size of 300x300, but with a border of 50px around the original image of 200px
- On the page, the displayed image has a size of 150px (this could be for two reasons (1) high definition display (2) responsive image resizing) 

With Prestashop default behavior, we will now see an image of 100px with a white border of 25px around.  😭 
With this setting enabled, we will se the image taking all the 150px available. 😄